### PR TITLE
fix: make the request T/O configurable

### PR DESCRIPTION
**Pull request by efimk-lu**
I also increased the default T/O from 250 to 300, WDYT?

### CODE REVIEW

**Comment by efimk-lu** [comment]
@doriaviram any specific way I should test it or the IT are enough?

File: src/httpSpansAgent.js [hunk 1]
```diff
@@ -11,8 +11,11 @@ import {
 } from './utils';
 import axios from 'axios';
 
-const REQUEST_TIMEOUT = 250;
-
+const REQUEST_TIMEOUT_FLAG_MS = 'LUMIGO_REQUEST_TIMEOUT_MS';
+const getRequestTimeout = () => {
+  if (process.env[REQUEST_TIMEOUT_FLAG_MS]) return parseInt(process.env[REQUEST_TIMEOUT_FLAG_MS]);
+  return 300;
+};
```
Comment (doriaviram) [inline src/httpSpansAgent.js:18]: Nice, move to `utils.ts`

File: src/httpSpansAgent.test.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ describe('HttpSpansAgent', () => {
       Authorization: 't_xxx',
       'User-Agent': '@lumigo/tracerMock$1.2.3',
     });
-    expect(requests[0].timeout).toEqual(250);
+    expect(requests[0].timeout).toEqual(300);
```
Comment (doriaviram) [inline src/httpSpansAgent.test.js:48]: Add a test that its configured by env var

**Review by doriaviram — APPROVED** [review]

**Pull request merged**